packages/block-editor/src/components/iframe/index.js

```diff
@@ -176,7 +176,7 @@ function Iframe(
 			const { contentDocument, ownerDocument } = node;
 			const { readyState, documentElement } = contentDocument;
 
-			if ( readyState !== 'interactive' && readyState !== 'complete' ) {
+			if ( readyState !== 'complete' && readyState !== 'interactive' ) {
```

Comment:
In the end this part of the patch only swaps the order of the two `readyState` checks and is redundant.